src/main/index.ts

```diff
@@ -432,3 +425,9 @@ async function forwardActions (
   mainStore.dispatch(message.action, message.sourceId);
   return defaultResponse();
 }
+
+async function saveOnClose() {
+  mainStore.dispatch(dimMenuAction(true));
```

Comment:
We are better to dim the whole application, so user is not able to keep interacting with it [_in event of enormously long disk writes_].